pep-0458-tuf-online-keys.rst

```diff
@@ -395,6 +395,208 @@ specifying `delegations`__.
+for the root role, is done by changing the keys listed for the roles in
+this file.
+
+The format of root.json is as follows:
+
+{
+  "_type" : "Root",
+  "version" : VERSION,
+  "expires" : EXPIRES,
+  "keys" : {
+    KEYID : KEY
+    , ... },
+  "roles" : {
+    ROLE : {
+      "keyids" : [ KEYID, ... ] ,
+      "threshold" : THRESHOLD },
+    ...
+  }
+}
+
+VERSION is an integer that is greater than 0.  Clients MUST NOT replace a
+metadata file with a version number less than the one currently trusted.
+
+EXPIRES determines when metadata should be considered expired and no longer
+trusted by clients.  Clients MUST NOT trust an expired file.
+
+A ROLE is one of "root", "snapshot", "targets", "timestamp", or "mirrors".
+A role for each of "root", "snapshot", "timestamp", and "targets" MUST be
+specified in the key list. The role of "mirror" is optional.  If not
+specified, the mirror list will not need to be signed if mirror lists are
+being used.
+
+The KEYID must be correct for the specified KEY.  Clients MUST calculate
+each KEYID to verify this is correct for the associated key.  Clients MUST
+ensure that for any KEYID represented in this key list and in other files,
+only one unique key has that KEYID.
+
+The THRESHOLD for a role is an integer of the number of keys of that role
+whose signatures are required in order to consider a file as being properly
+signed by that role.
+
+Metadata date-time data follows the ISO 8601 standard.  The expected format
+of the combined date and time string is "YYYY-MM-DDTHH:MM:SSZ".  Time is
+always in UTC, and the "Z" time zone designator is attached to indicate a
+zero UTC offset.  An example date-time string is "1985-10-21T01:21:00Z".
+
+
+snapshot.JSON
+~~~~~~~~~~~~~
+
+The snapshot.json file is signed by the snapshot role.  It lists hashes and
+sizes of all metadata on the repository, excluding timestamp.json and
+mirrors.json.
+
+The format of snapshot.json is as follows:
+
+{
+  "_type" : "Snapshot",
+  "version" : VERSION,
+  "expires" : EXPIRES,
+  "meta" : METAFILES
+}
+
+METAFILES is an object whose format is the following:
+
+{
+  METAPATH : {
+    "length" : LENGTH,
+    "hashes" : HASHES,
+    ("custom" : { ... }) },
+  ...
+}
+
+METAPATH is the the metadata file's path on the repository relative to the
+metadata base URL.
+
+
+targets.JSON and delegated target roles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The targets.json metadata file lists the hashes and sizes of target files.
+Target files are the actual files that clients are intending to download (for
+example, the software updates they are trying to obtain).
+
+This file can optionally define other roles to which it delegates trust.
+Delegating trust means that the delegated role is trusted for some or all of
+the target files available from the repository. When delegated roles are
+specified, they are specified in a similar way to how the Root role specifies
```

Comment:
I would change this "they are specified in a similar way to how" to this: "it is done similar to how"